internal/backend/db/dbgorm/sessions_test.go

```diff
@@ -411,7 +411,7 @@ func TestSessionLogRecordNextPageTokenNotEmpty(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, len(res.Log.Records()), 2)
 	assert.Equal(t, res.TotalCount, int64(3))
-	assert.Equal(t, res.PaginationResult.NextPageToken, "2")
+	assert.NotEmpty(t, res.NextPageToken)
```

Comment:
It's not perfect for highly concurrent scale, 
but since we limit it to a specific session, 
concurrency is basically not an option (except for the current bug we have with python prints)
so it should be ok

also, this is related to session log records, and we are not a high scale system in the workflow sense, so it's not an issue and we can take this assumption for now